tests/test_autorift.py

```diff
@@ -89,16 +89,16 @@ def test_golden_products(comparison_environments, job_name, user_id, keep):
             continue
 
         main_nc = main_job.download_files(main_dir)[0]
-        main_hash = main_nc.stem
-
         develop_nc = develop_job.download_files(develop_dir)[0]
-        develop_hash = develop_nc.stem
+        if keep:  # always used in local testing
+            _ = hyp3_sdk.util.download_file(main_job.browse_images[0], main_nc.with_suffix('.png'))
+            _ = hyp3_sdk.util.download_file(develop_job.browse_images[0], develop_nc.with_suffix('.png'))
```

Comment:
Or was the `_ =` bit to satisfy a linter warning?
```suggestion
            hyp3_sdk.util.download_file(main_job.browse_images[0], main_nc.with_suffix('.png'))
            hyp3_sdk.util.download_file(develop_job.browse_images[0], develop_nc.with_suffix('.png'))
```

Reply:
Yeah, that `_ = ` is the convention that tells the linter "I don't care about the return value"